packages/labs/scoped-registry-mixin/README.md

```diff
@@ -0,0 +1,43 @@
+# @lit-labs/scoped-registry-mixin
+
+`UseScopedRegistry` mixin for `LitElement` that integrates with the speculative [Scoped CustomElementRegistry polyfill](https://github.com/webcomponents/polyfills/tree/master/packages/scoped-custom-element-registry).
```

Comment:
"Use" just sounds a lot like React hooks to me now. Is there another name that works?

Reply:
what about `withScopedRegistry`?

Reply:
I prefer mixins to be named like classes with capitalized noun phrases, since it represents a type like a class. I'd suggest something like `ScopedRegistryHost`.

Reply:
oh, right, my mistake. I agree. I prefer capitalized names for mixins too.

`ScopedRegistryHost` looks good to me.

Reply:
done